meson.build

```diff
@@ -92,6 +92,16 @@ install_data(
 	install_mode: 'rwxr-xr-x',
 )
 
+conf_data = configuration_data()
+conf_data.set('prefix', get_option('prefix'))
```

Comment:
Can't we use `bindir` here?

Reply:
How about:
```
conf_data.set('bindir', join_paths(get_option('prefix'), get_option('bindir')))
```
And in fr.emersion.mako.service.in:
```
[D-BUS Service]
Name=org.freedesktop.Notifications
Exec=@bindir@/mako
```